modules/apps/jaggery/emm/src/emm/units/user-listing/public/js/user-listing.js

```diff
@@ -49,10 +49,10 @@ function hidePopup() {
  */
 function getSelectedUsernames() {
     var usernameList = [];
-    var thisTable = $(".DTTT_selected").closest('.dataTables_wrapper').find('.dataTable').dataTable();
-    thisTable.api().rows().every(function(){
-        if($(this.node()).hasClass('DTTT_selected')){
-			usernameList.push($(thisTable.api().row(this).node())[0].getAttribute('data-username'));
+    var userList = $("#user-grid").find('> tbody > tr');
+    userList.each(function() {
+        if($(this).hasClass('DTTT_selected')){
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158